src/multibit_fmt_plug.c

```diff
@@ -321,6 +361,20 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 				if (is_bitcoinj_protobuf_data(outbuf))
 					cracked[index] = 1;
 			}
+		} else if (cur_salt->type == 3) {
+			UTF16 password[PLAINTEXT_LENGTH * 2 + 1];
+
+			len = enc_to_utf16_be(password, PLAINTEXT_LENGTH, (const unsigned char*)saved_key[index], len + 1);
```

Comment:
You should probably set FMT_UNICODE as well as FMT_UTF8!

Reply:
Done now. Thanks!

I will try to add some Unicode test vectors.